llarp/net/net_addr.cpp

```diff
@@ -1,505 +0,0 @@
-#include <net/net.hpp>
-#include <net/net_addr.hpp>
-#include <string_view>
-
-// for addrinfo
-#ifndef _WIN32
-#include <sys/types.h>
-#include <sys/socket.h>
-#include <netdb.h>
-#else
-#include <winsock2.h>
-#include <ws2tcpip.h>
-#include <wspiapi.h>
-#define inet_aton(x, y) inet_pton(AF_INET, x, y)
```

Comment:
@notlesh you forgot to retain these

Reply:
Thanks for pointing that out, it's hard to catch with `#ifdef`. I intend to migrate some of this stuff to `SockAddr`, so I'll keep this in mind.